docs/getting-started-guides/kubeadm.md

```diff
@@ -0,0 +1,245 @@
+
+By way of example, you can install [Weave Net](https://github.com/weaveworks/weave-kube) by logging in to the master and running:
+
+    # kubectl apply -f https://git.io/weave-kube
+    daemonset "weave-net" created
+
+If you prefer [Calico](https://github.com/projectcalico/calico-containers/tree/master/docs/cni/kubernetes/manifests/kubeadm) or [Canal](https://github.com/tigera/canal/tree/master/k8s-install/kubeadm), please refer to their respective installation guides.
+You should only install one pod network per cluster.
+
+Once a pod network has been installed, you can confirm that it is working by checking that the `kube-dns` pod is `Running` in the output of `kubectl get pods --all-namespaces`.
+**This signifies that your cluster is ready.**
+
+### (Optional) Installing a sample application
+
+As an example, install a sample microservices application, a socks shop, to put your cluster through its paces.
+To learn more about the sample microservices app, see the [GitHub README](https://github.com/microservices-demo/microservices-demo).
+
+    # git clone https://github.com/microservices-demo/microservices-demo
+    # kubectl apply -f microservices-demo/deploy/kubernetes/manifests
+
+You can then find out the port that the [NodePort feature of services](/docs/user-guide/services/) allocated for the front-end service by running:
+
+    # kubectl describe svc front-end
+    Name:                   front-end
+    Namespace:              default
+    Labels:                 name=front-end
+    Selector:               name=front-end
+    Type:                   NodePort
+    IP:                     100.66.88.176
+    Port:                   <unset> 80/TCP
+    NodePort:               <unset> 31869/TCP
+    Endpoints:              <none>
+    Session Affinity:       None
+
+It takes several minutes to download and start all the containers, watch the output of `kubectl get pods` to see when they're all up and running.
+
+Then go to the IP address of your cluster's master node in your browser, and specify the given port.
+So for example, `http://<master_ip>:<port>`.
+In the example above, this was `31869`, but it is a different port for you.
+
+If there is a firewall, make sure it exposes this port to the internet before you try to access it.
+
+### Explore other add-ons
+
+See the [list of add-ons](/docs/admin/addons/) to explore other add-ons, including tools for logging, monitoring, network policy, visualization &amp; control of your Kubernetes cluster.
+
+
+## What's next
+
+* Learn more about [Kubernetes concepts and kubectl in Kubernetes 101](/docs/user-guide/walkthrough/).
+* Install Kubernetes with [a cloud provider configurations](/docs/getting-started-guides/) to add Load Balancer and Persistent Volume support.
+
+
+## Cleanup
+
+* To uninstall the socks shop, run `kubectl delete -f microservices-demo/deploy/kubernetes/manifests` on the master.
+* To uninstall Kubernetes, simply delete the machines you created for this tutorial.
```

Comment:
I have a script for this, which we should refer to. Saying delete your machines is kind of awful :smile:
```shell
systemctl stop kubelet; 
docker rm -f $(docker ps -q); mount | grep "/var/lib/kubelet/*" | awk '{print $3}' | xargs umount 1>/dev/null 2>/dev/null; 
rm -rf /var/lib/kubelet /etc/kubernetes /var/lib/etcd /etc/cni; 
ip link set cbr0 down; ip link del cbr0; 
ip link set cni0 down; ip link del cni0;
systemctl start kubelet; 
```